safe/gis/vector/tools.py

```diff
@@ -324,6 +324,47 @@ def read_dynamic_inasafe_field(inasafe_fields, dynamic_field, black_list=None):
     return unique_exposure
 
 
+def measure_bearing(point_a, point_b):
+    """Measure the bearing angle between two points.
+
+    :param point_a: First Point.
+    :type point_a: QgsPoint
+
+    :param point_b: Second Point.
+    :type point_b: QgsPoint
+
+    :return: The distance between input points.
+    :rtype: float
+    """
+    return point_b.azimuth(point_a)
```

Comment:
Maybe you can remove this function and call the azimuth straight in the parent function? It like more docstring than code :)